homeassistant/components/mqtt/client.py

```diff
@@ -659,6 +676,18 @@ async def async_connect(self, client_available: asyncio.Future[bool]) -> None:
                     self.conf[CONF_BROKER],
                     self.conf.get(CONF_PORT, DEFAULT_PORT),
                     self.conf.get(CONF_KEEPALIVE, DEFAULT_KEEPALIVE),
+                    "",  # bind_address default
```

Comment:
The arg list is getting rather long, consider using a partial here so kwargs can be used

Reply:
Replaced args with a partial with kwargs. Thanks!